ci/test/04_install.sh

```diff
@@ -98,6 +91,16 @@ elif [ "$TRAVIS_OS_NAME" != "osx" ]; then
   ${CI_RETRY_EXE} DOCKER_EXEC apt-get install --no-install-recommends --no-upgrade -y $PACKAGES $DOCKER_PACKAGES
 fi
 
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+  top -l 1 -s 0 | awk ' /PhysMem/ {print}'
+  echo "Number of CPUs: $(sysctl -n hw.logicalcpu)"
+else
+  DOCKER_EXEC free -m -h
+  DOCKER_EXEC echo "Number of CPUs \(nproc\):" \$\(nproc\)
+  DOCKER_EXEC echo "Free disk space:"
+  DOCKER_EXEC df -h
+fi
+
```

Comment:
17767 why not add the `wrap-qemu` file? and changes in `ci/test/06_script_b.sh`

Reply:
we don't have all of this code that is refactored to `wrap-qemu`.
These changes are missing since the are not included within old backport `Merge #17041: ci: Run tests on arm`:
```
+if [ -n "$QEMU_USER_CMD" ]; then
+  BEGIN_FOLD wrap-qemu
+  echo "Prepare to run functional tests for HOST=$HOST"
+  # Generate all binaries, so that they can be wrapped
+  DOCKER_EXEC make $MAKEJOBS -C src/secp256k1 VERBOSE=1
+  DOCKER_EXEC make $MAKEJOBS -C src/univalue VERBOSE=1
+  for b_name in {"${BASE_OUTDIR}/bin"/*,src/secp256k1/*tests,src/univalue/{no_nul,test_json,unitester,object}}; do
+    # shellcheck disable=SC2044
+    for b in $(find "${BASE_ROOT_DIR}" -executable -type f -name $(basename $b_name)); do
+      echo "Wrap $b ..."
+      DOCKER_EXEC mv "$b" "${b}_orig"
+      DOCKER_EXEC echo "\#\!/usr/bin/env bash" \> "$b"
+      DOCKER_EXEC echo "$QEMU_USER_CMD \\\"${b}_orig\\\" \\\"\\\$@\\\"" \>\> "$b"
+      DOCKER_EXEC chmod +x "$b"
+    done
+  done
+  END_FOLD
+fi
```

The reason why this changes are missing, because we don't use `ci/test_run_all.sh` that build all code for all platform and runs all functional/unit tests such as tsan/etc locally.

I tried to run this script and it seems as it broken currently.
I can try to fix `ci/test_run_all.sh` but it seems IMO as out of scope this PR. I will try to fix `ci/test_run_all.sh` later

Reply:
```
$ ci/test_run_all.sh 
Setting specific values in env
Fallback to default values in env (if not yet set)
CCACHE_TEMPDIR=/tmp/.ccache-temp
TSAN_OPTIONS=suppressions=/test/sanitizer_suppressions/tsan
CCACHE_SIZE=100M
CCACHE_DIR=/home/knst/projects/dash-reviews/ci-cache-linux64/ccache
CCACHE_COMPRESS=1
ASAN_OPTIONS=detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1
LSAN_OPTIONS=suppressions=/test/sanitizer_suppressions/lsan
BOOST_TEST_RANDOM=1
UBSAN_OPTIONS=suppressions=/test/sanitizer_suppressions/ubsan:print_stacktrace=1:halt_on_error=1:report_error_type=1
WINEDEBUG=fixme-all
LC_ALL=C.UTF-8
Creating ubuntu:focal container to run in
focal: Pulling from library/ubuntu
ca1778b69356: Pull complete 
Digest: sha256:db8bf6f4fb351aa7a26e27ba2686cf35a6a409f65603e59d4c203e58387dc6b3
Status: Downloaded newer image for ubuntu:focal
docker.io/library/ubuntu:focal
              total        used        free      shared  buff/cache   available
Mem:           38Gi       3.7Gi        32Gi       135Mi       2.8Gi        34Gi
Swap:            0B          0B          0B
Number of CPUs (nproc): 16
Free disk space:
Filesystem      Size  Used Avail Use% Mounted on
overlay          96G   17G   75G  18% /
tmpfs            64M     0   64M   0% /dev
shm              64M     0   64M   0% /dev/shm
/dev/nvme0n1p8  1.5T  527G  909G  37% /ro_base
/dev/nvme0n1p7   96G   17G   75G  18% /etc/hosts
tmpfs            20G     0   20G   0% /proc/asound
tmpfs            20G     0   20G   0% /proc/acpi
tmpfs            20G     0   20G   0% /proc/scsi
tmpfs            20G     0   20G   0% /sys/firmware
Error: No such container: bash
Before retry #1: sleeping 0.3 seconds
Error: No such container: bash
...
```
current failure when running `ci/test_run_all.sh`